workitem/link/link_repository.go

```diff
@@ -111,6 +113,10 @@ func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targe
 		}
 		return nil, errors.NewInternalError(db.Error.Error())
 	}
+	// save a revision of the created work item link
+	if err := r.revisionRepo.Create(ctx, creatorID, RevisionTypeCreate, *link); err != nil {
+		return nil, errs.WithStack(err)
```

Comment:
Please use `errs.Wrapf(err, "")` for newly added errors like this to give some context.